packages/shared/src/graphql/notifications.ts

```diff
@@ -83,3 +83,11 @@ export const READ_NOTIFICATIONS_MUTATION = gql`
     }
   }
 `;
+
+export const NEW_NOTIFICATIONS_SUBSCRIPTION = gql`
+  subscription NewNotification {
+    newNotifications {
```

Comment:
@idoshamun I think this should be singular, it only returns 1 notification

Reply:
I was also considering whether it should be singular or plural. The stream will send you all the new notifications, but you will get them one by one. So it depends on the perspective. Anyway, I'll be happy to change it if it makes more sense from your perspective.

Reply:
I usually name functions based on their return types. If it returns an array, it is plural, otherwise, it is singular.

Reply:
Let me change that quickly

Reply:
Deploying now

Reply:
@idoshamun I have updated the function here to `newNotification`

Reply:
Let me know when it's ready for review :)